### PR TITLE
Anopheles gambiae species complex chromosomal inversion data items

### LINKED ISSUE #117: NTR: Anopheles gambiae species complex chromosomal inversion data items
Analogous to the Kdr L1014F-style variables we made recently I would like to represent chromosomal inversion genotypes in the same way: one assay has multiple variables to describe the genotype. 

These are the inversions we need - one on the 2L chromosome arm, and the others on 2R.

-  2La https://legacy.vectorbase.org/Anopheles_gambiae/StructuralVariation/Explore?sv=2La;svf=1;vdb=variation
-  2Rb https://legacy.vectorbase.org/Anopheles_gambiae/StructuralVariation/Explore?sv=2Rb;svf=1;vdb=variation
-  2Rc https://legacy.vectorbase.org/Anopheles_gambiae/StructuralVariation/Explore?sv=2Rc;svf=1;vdb=variation
-  2Rd https://legacy.vectorbase.org/Anopheles_gambiae/StructuralVariation/Explore?sv=2Rd;svf=1;vdb=variation
-  2Rj https://legacy.vectorbase.org/Anopheles_gambiae/StructuralVariation/Explore?sv=2Rj;svf=1;vdb=variation
-  2Ru https://legacy.vectorbase.org/Anopheles_gambiae/StructuralVariation/Explore?sv=2Ru;svf=1;vdb=variation
-  2Rk Not in VectorBase Ensembl

There are some existing ontology terms out there, e.g. in IDOMAL/MIRO
https://www.ebi.ac.uk/ols/ontologies/idomal/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMIRO_30000074&viewMode=All&siblings=true
But these don't have definitions...  however, the Ensembl genome browser links above provide reasonable information, including approx chromosomal locations.

The variables will store an integer number of copies of the inverted form, so 0, 1, or 2 (because the mosquito is diploid), so a label for the variables could be something like "Number of copies of inverted allele of 2La chromosomal inversion" - though let's aim for something much much shorter!

### PR DESCRIPTION
**Pull request by johnwjudkins**
Closes #117

### CODE REVIEW

**Review by zhengj2007 — COMMENTED** [review]
Would be better to add 'Term editor' and 'Definition source' annotations.

**Review by cstoeckert — CHANGES_REQUESTED** [review]
- labels don't read correctly with change: 'number of 2La Anopheles gambiae complex chromosomal inversion' -> inversions
- textual definition for 2rj is wrong ( says 2LA)
- 2Rd editor note should be 2Rd not 2Rj

File: src/ontology/eupath_dev.owl [hunk 1]
```diff
@@ -16452,6 +16452,90 @@ CD3T188G is a polymophism on the human CD36 gene.</obo:IAO_0000116>
...
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/EUPATH_0043217 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/EUPATH_0043217">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/STATO_0000047"/>
+        <obo:IAO_0000115>A count of the number of copies of the chromosomal inversion 2Rd in Anopheles gambiae.</obo:IAO_0000115>
+        <obo:IAO_0000116>2Rj is described in: George, P., Sharakhova, M.V. and Sharakhov, I.V., 2010. High‐resolution cytogenetic map for the African malaria vector Anopheles gambiae. Insect molecular biology, 19(5), pp.675-682. PMID:20609021</obo:IAO_0000116>
```
Comment (bobular) [inline src/ontology/eupath_dev.owl:16520]: Just noticed that this should be "2Rd is described in"

The citation is correct.

(Oh, I see Chris already saw this.)

File: src/ontology/eupath_dev.owl [hunk 2]
```diff
@@ -16452,6 +16452,90 @@ CD3T188G is a polymophism on the human CD36 gene.</obo:IAO_0000116>
...
 
 
+    <!-- http://purl.obolibrary.org/obo/EUPATH_0043212 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/EUPATH_0043212">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/STATO_0000047"/>
+        <obo:IAO_0000115>A count of the number of copies of the interspecific chromosomal inversion (2La) in Anopheles gambiae complex.</obo:IAO_0000115>
+        <obo:IAO_0000116>2La is located on chromosome arm 2L:20524058-42165532 (approximate location in the AgamP3 assembly; GenBank: GCF_000005575.2) as described in: Sharakhov, I.V., White, B.J., Sharakhova, M.V., Kayondo, J., Lobo, N.F., Santolamazza, F., Della Torre, A., Simard, F., Collins, F.H. and Besansky, N.J., 2006. Breakpoint structure reveals the unique origin of an interspecific chromosomal inversion (2La) in the Anopheles gambiae complex. Proceedings of the National Academy of Sciences, 103(16), pp.6258-6262. PMID: 16606844</obo:IAO_0000116>
+        <obo:IAO_0000233>https://github.com/VEuPathDB-ontology/VEuPathDB-ontology/issues/117</obo:IAO_0000233>
+        <rdfs:label>number of 2La Anopheles gambiae complex chromosomal inversion</rdfs:label>
```
Comment (bobular) [inline src/ontology/eupath_dev.owl:16462]: It's tricky without adding a lot more words:

Number of copies of the 2La Anopheles gambiae complex chromosomal inversion

File: src/ontology/eupath_dev.owl (hunk 1 above)
Comment (johnwjudkins) [reply src/ontology/eupath_dev.owl]: Resolved in "Fixed texual definition & editor note" commit.

File: src/ontology/eupath_dev.owl (hunk 2 above)
Comment (cstoeckert) [reply src/ontology/eupath_dev.owl:16462]: Agree with Bob's suggested form of label.

**Review by cstoeckert — APPROVED** [review]

**Review by zhengj2007 — APPROVED** [review]

File: src/ontology/eupath_dev.owl [hunk 3]
```diff
@@ -16452,6 +16452,104 @@ CD3T188G is a polymophism on the human CD36 gene.</obo:IAO_0000116>
...
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/EUPATH_0043217 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/EUPATH_0043217">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/STATO_0000047"/>
+        <obo:IAO_0000115>A count of the number of copies of the chromosomal inversion 2Rd in Anopheles gambiae.</obo:IAO_0000115>
+        <obo:IAO_0000116>2Rd is described in: George, P., Sharakhova, M.V. and Sharakhov, I.V., 2010. High‐resolution cytogenetic map for the African malaria vector Anopheles gambiae. Insect molecular biology, 19(5), pp.675-682. PMID:20609021</obo:IAO_0000116>
```
Comment (bobular) [inline src/ontology/eupath_dev.owl:16530]: Missing the location info
```suggestion
        <obo:IAO_0000116>2Rd is located on chromosome arm 2R:31480000-42600000 (location in the AgamP3 assembly; GenBank: GCF_000005575.2) as described in: George, P., Sharakhova, M.V. and Sharakhov, I.V., 2010. High‐resolution cytogenetic map for the African malaria vector Anopheles gambiae. Insect molecular biology, 19(5), pp.675-682. PMID:20609021</obo:IAO_0000116>
```

File: src/ontology/eupath_dev.owl [hunk 4]
```diff
@@ -16452,6 +16452,104 @@ CD3T188G is a polymophism on the human CD36 gene.</obo:IAO_0000116>
...
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/EUPATH_0043216 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/EUPATH_0043216">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/STATO_0000047"/>
+        <obo:IAO_0000115>A count of the number of copies of the trans-specific inversion 2Ru in Anopheles gambiae.</obo:IAO_0000115>
+        <obo:IAO_0000116>2Ru is described in: White, B.J., Cheng, C., Sangaré, D., Lobo, N.F., Collins, F.H. and Besansky, N.J., 2009. The population genomics of trans-specific inversion polymorphisms in Anopheles gambiae. Genetics, 183(1), pp.275-288. PMID:19581444</obo:IAO_0000116>
```
Comment (bobular) [inline src/ontology/eupath_dev.owl:16516]: Missing location info
```suggestion
        <obo:IAO_0000116>2Ru is located on chromosome arm 2R:31480000-35500000 (location in the AgamP3 assembly; GenBank: GCF_000005575.2) as described in: White, B.J., Cheng, C., Sangaré, D., Lobo, N.F., Collins, F.H. and Besansky, N.J., 2009. The population genomics of trans-specific inversion polymorphisms in Anopheles gambiae. Genetics, 183(1), pp.275-288. PMID:19581444</obo:IAO_0000116>
```

**Review by bobular — CHANGES_REQUESTED** [review]
Thanks John - I just spotted two missing locations - or did you exclude those for a reason?

**Review by bobular — APPROVED** [review]

**Pull request merged**